build.gradle.kts

```diff
@@ -44,6 +44,6 @@ tasks.register<JavaExec>("ktlintFormat") {
 
 tasks.wrapper {
     distributionSha256Sum =
-        URL("$distributionUrl.sha256")
+        URI.create("$distributionUrl.sha256").toURL()
```

Comment:
URL constructora are deprecated in Java 20: 
>The java.net.URL constructors are deprecated. Developers are encouraged to use [java.net.URI](https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/URI.html) to parse or construct a URL.

https://docs.oracle.com/en/java/javase/20/docs/api/java.base/java/net/URL.html#%3Cinit%3E(java.lang.String